docs/android-frontend-testing.md

```diff
@@ -97,13 +97,13 @@ These fields affect which OS, app and build the experiment is for. These should
 
 These fields affect segment the population for eligibility for the experiment, and which branch they'll be given.
 
- * `targeting`: Experimenter will help you generate this JEXL query string. If Nimbus evaluates this on a given device to `true`, then the device is eligible for the experiment. If your testing the app, then you 
+ * `targeting`: Experimenter will help you generate this JEXL query string. If Nimbus evaluates this on a given device to `true`, then the device is eligible for the experiment. If your testing the app, then you (TODO)
```

Comment:
Notice this looks like a partial thought. Once/if this lands I can file a new issue in mozilla/experimenter to add more content.